### PR TITLE
aboutpage

### PR DESCRIPTION
**Pull request by moatazalaa14**

### CODE REVIEW

File: src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    }
+
+    .content-AZAIEEE  {
+        display: grid;
+        grid-template-columns: 1fr;
+        grid-auto-rows: minmax(40vh , auto);
+        justify-items: center;
+    }
+
+    .IEEEAZALogo img{
```
Comment (dvmhmdsd) [inline src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.css]: Use `BEM` methodology to name classes.

File: src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+}
+.content-AZAIEEE  {
+    display: grid;
+    grid-template-columns: repeat(2,1fr);
+    grid-auto-rows: minmax(40vh , auto);
+    align-items: center;
+}
+
+.content-AZAIEEE > *{
+    width: 45vw;
```
Comment (dvmhmdsd) [inline src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.css]: Don't add fixed width

File: src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+}
+
+.IEEEAZALogo img{
+    width: 40vw;
```
Comment (dvmhmdsd) [inline src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.css]: Don't add fixed width

File: src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useRef } from "react";
+import "./IEEEAZAHARSec.css";
+import IEEEAZALogo from "./../../../../../../../assets/IEEE-pub-logo.jpg";
+function IEEEAZAHARSec() {
+  const AZAlogo = useRef();
```
Comment (dvmhmdsd) [inline src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.js]: Don't use hooks.

File: src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  useEffect(() => {
```
Comment (dvmhmdsd) [inline src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.js]: Don't use hooks

File: src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+      if (entry.isIntersecting === true) {
+        AZAlogo.current.innerHTML = `<img src=${IEEEAZALogo} / >`;
+      }
+    });
+    observer.observe(AZAlogo.current);
+  });
+  return (
+    <>
+      <section className="AZA-IEEE-sec">
+        <h2>IEEE-AZAHAR</h2>
```
Comment (dvmhmdsd) [inline src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.js]: Why did u use `h2` here? is this the right place ? (semantic)

File: src/modules/static-pages/about/components/page/components/headerForAboutPage/HeaderForAboutPage.css [hunk 7]
```diff
@@ -0,0 +1,18 @@
+#header-about{
```
Comment (dvmhmdsd) [inline src/modules/static-pages/about/components/page/components/headerForAboutPage/HeaderForAboutPage.css]: Use classes not ids in styling

File: src/modules/static-pages/about/components/page/components/publicIEEESec/PublicIEEESec.js [hunk 8]
```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useRef } from "react";
+import "./PublicIEEESec.css";
+import IEEEPubLogo from "./../../../../../../../assets/IEEE-pub-logo.jpg";
```
Comment (dvmhmdsd) [inline src/modules/static-pages/about/components/page/components/publicIEEESec/PublicIEEESec.js]: Why did u rename the image?

File: src/modules/static-pages/about/components/page/index.js [hunk 9]
```diff
@@ -1,7 +1,20 @@
...
+import HeaderForAboutPage from "./components/headerForAboutPage/HeaderForAboutPage";
+import PublicIEEESec from "./components/publicIEEESec/PublicIEEESec";
+import IEEEAZAHARSec from "./components/IEEEAZAHARSec/IEEEAZAHARSec";
+import IEEEStructure from "./components/IEEEStructure/IEEEStructure";
+import Layout from "./../../../../../shared/layout/index";
+import Footer from "./../../../../../shared/footer/index";
+function AboutPage() {
+  return (
+    <>
+      <Layout />
```
Comment (dvmhmdsd) [inline src/modules/static-pages/about/components/page/index.js]: Use `Layout` like this :
```
<Layout>
 <HeaderForAboutPage />
      <PublicIEEESec />
      <IEEEAZAHARSec />
      <IEEEStructure />
</Layout>
```

and remove the footer, it's added already inside the `Layout` component.

File: src/modules/static-pages/about/components/page/style.module.css [hunk 10]
```diff
@@ -0,0 +1,4 @@
+
+.aboutpage-dark{
```
Comment (dvmhmdsd) [inline src/modules/static-pages/about/components/page/style.module.css]: You don't need to set the bg of the page in the dark mode, it has already set in the body.

**Review by dvmhmdsd — CHANGES_REQUESTED** [review]
1. Don't change the name of `css` files, leave it as `style.module.css` as it's and use `css modules` as noted in `README` file.

2. Use the `container` class from bootstrap to add side spaces.

3. Add the logo of the team not the `IEEE Club`.

4. Add support to dark mode.

5. Make the image life under paragraph in medium screen.

6. Add heading to the page --> h1

File: src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.css (hunk 3 above)
Comment (moatazalaa14) [reply src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.css]: ???i make it taking auto width ??

File: src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.js (hunk 6 above)
Comment (moatazalaa14) [reply src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.js]: > <h1> headings should be used for main headings, followed by <h2> headings, then the less important <h3>, and so on.
الي فهمته ان يكون في h1 بعد كدا ابدا ادرج فيهم 
فانا فاهم غلط ؟

File: src/modules/static-pages/about/components/page/components/publicIEEESec/PublicIEEESec.js (hunk 8 above)
Comment (moatazalaa14) [reply src/modules/static-pages/about/components/page/components/publicIEEESec/PublicIEEESec.js]: عشان الصور الي موجود فالصفحة كلها صور هيتحط عليها شروط فبدل ما اتعامل مع الpath الكبير دة
اتعامل مع الاسم

File: src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.css (hunk 3 above)
Comment (dvmhmdsd) [reply src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.css]: Yup

File: src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.js (hunk 6 above)
Comment (dvmhmdsd) [reply src/modules/static-pages/about/components/page/components/IEEEAZAHARSec/IEEEAZAHARSec.js]: انت كده فاهم كويس جدا بس ال 
h1
فين ؟

**Review by dvmhmdsd — COMMENTED** [review]
1. Don't change the name of the files , leave it `index.js` & `style.module.css`.

2. The names of components is not appropriate, and has typos e.g: `IEEEAZAHARSec` ==> `IEEEAZHARSection`

3. Class names are not descriptive.

4. Don't add fixed width to images.

5. Use the `IntersectionObserver` for the video only.

**Pull request merged**